packages/desktop-client/src/components/mobile/transactions/TransactionEdit.jsx

```diff
@@ -641,7 +638,7 @@ const TransactionEditInner = memo(function TransactionEditInner({
 
             const { account: accountId } = unserializedTransaction;
             if (accountId) {
-              navigate(`/accounts/${accountId}`, { replace: true });
+              navigate(-1);
```

Comment:
This is going to break an existing workflow where the app opens the account where new transaction is added to.

Reply:
For delete? I think you mean the other one above which is for saving.  When adding a new transaction it would 'back' instead of going to the accounts.  I see what you mean there.